### DISCUSSION TITLE
Windows machine as a routing peer

### DISCUSSION THREAD

**Discussion by grahamIT** [netbirdio/netbird#1438] [Q&A / Support]
Hello,

I am considering using Netbird in our environment as a alternative to port forwarding. However, I noticed Windows is not a option for routing peers. Would it be possible to add Windows support for this? If not is there a cheap single board computer that can be purchased in semi bulk to do traffic routing?

**Comment by pappz** [comment]
Hello @grahamIT,

Currently, we can support routing features on Linux operation systems only. From a technical point of view, it is not impossible to develop this feature for Windows but we do not have exact plans for it.

**Reply by grahamIT** [reply]
This in the one feature that Netbird is missing for me. I need to access NVRs remotely and all the machines on the local network are windows. I'm going to open a issue as I thing this feature needs more attention.

**Comment by grahamIT** [comment]
Does anyone have alternatives to this? I right now I am looking into just using Nginx Streams to forward traffic but I have a feeling there much be an easier way.
